test/cts/tool/CTSConverter/README.md

```diff
@@ -1,48 +1,57 @@
 # CTSConverter
-Transfer [nn](https://android.googlesource.com/platform/frameworks/ml) test case to [webml](https://github.com/intel/webml-polyfill) test case
+Transfer [nn (tag: android-cts-10.0_r2)](https://android.googlesource.com/platform/frameworks/ml/+/refs/tags/android-cts-10.0_r2) test case to [webml](https://github.com/intel/webml-polyfill) test case
 
 ## Prerequisites
 * Need `python3`
 
 ## Start
 
-### create `cts-all.js` into `./output`, this file include nn and plus test case
+Transfer nn test case into `./output/cts`
 
 ```shell
 $ npm start
 ```
 
-### get more information for command line
+## Update or create combine test file
```

Comment:
@cuiyanx Please modify ‘Update or create combine test file’ to be ‘Update or create combined test files’.